network/twind/README

```diff
@@ -0,0 +1,60 @@
+twind is a simple daemon serving static files over the gemini protocol.
+
+It is intended to have as few knobs as possible and has no support for
+a configuration file.
+
+twind is named after the latin word for gemini - twins.
+
+Features:
+ * Serving static gemini files
+ * Virtual hosts
+ * MIME handling
+ * IPv4 and IPv6 support
+
+It doesn't support CGI handling and probably never will. There are more
+advanced gemini servers out there if you look for fancy stuff.
+
+---
+
+twind needs a dedicated user called '_twind' and directory to run.
+User will be created with SlackBuild script automatically: 'make user'
```

Comment:
It should be done manually by the user before the script can continue
please have a look on the example of other scripts in SBo repository